src/sass/_overlay_panel.scss

```diff
@@ -0,0 +1,25 @@
+.overlay-panel {
+  // mobile
+  height: 100vh;
+  padding-bottom: 0.75rem;
+  padding-top: 0.5rem;
+  position: absolute;
+  width: 100vw;
```

Comment:
Better rely on the vanilla width?

Reply:
Sure, but let's maybe change this within the permissions PR since these styles might change still

Reply:
How about having just the default styling here in this pr and adding any custom styling in the other PR if we need it?
So this whole file becomes:
```
.overlay-panel {
  height: 100vh;
  position: absolute;
  z-index: 103 !important;
}
```

Because the other spacings are kind of custom tailored to the permissions already and we should def. use vanilla variables instead of the hardcoded rem values.